controllers/clusters/postgresql_controller.go

```diff
@@ -286,6 +286,10 @@ func (r *PostgreSQLReconciler) handleCreateCluster(
 		return models.ReconcileRequeue
 	}
 
+	if pg.Annotations[models.ResourceStateAnnotation] == models.UpdatingEvent {
+		return models.ReconcileRequeueNow
+	}
```

Comment:
why do you need to go straight to update?

Reply:
It was needed when we used apiv1, to set twofactordelete on creation. Removed